### ISSUE TITLE
fatal error: module 'GoogleMaps' not found

### ISSUE THREAD

**Issue by Alaaeldin-as1405124** [nesterapp/react-native-streetview#31]
@import GoogleMaps;
 ~~~~~~~^~~~~~~~~~
1 error generated.

I'm struggling with this issue, tried so many solutions and nothing worked. I have installed google maps, tried to include GoogleMaps header by different ways but it still doesn't build

Notes:
- Tried to change Framework search paths multiple times
- Tried to import <GoogleMaps.h> && <GoogleMaps/GoogleMaps.h>
- Tried to clean the project and rebuild
- Tried to delete Pod.lock && Pod folder + clean + reinstall pods


Pod file:

platform :ios, '9.0'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

target 'x' do
  # Pods for x
  pod 'React', :path => '../node_modules/react-native/'
  pod 'React-Core', :path => '../node_modules/react-native/React'
  pod 'React-DevSupport', :path => '../node_modules/react-native/React'
  pod 'React-RCTActionSheet', :path => '../node_modules/react-native/Libraries/ActionSheetIOS'
  pod 'React-RCTAnimation', :path => '../node_modules/react-native/Libraries/NativeAnimation'
  pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
  pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
  pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
  pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
  pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
  pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
  pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
  pod 'React-RCTWebSocket', :path => '../node_modules/react-native/Libraries/WebSocket'

  pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
  pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
  pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
  pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
   pod 'PanoramaView', :path => '../node_modules/@lightbase/react-native-panorama-view'


  pod 'react-native-google-maps', :path => '../node_modules/react-native-maps'


  pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'

    source 'https://github.com/CocoaPods/Specs.git'
    pod 'GoogleMaps'
    pod 'GooglePlaces'


  pod 'react-native-streetview', :path => '../node_modules/react-native-streetview'

  target 'x-Tests' do
    inherit! :search_paths
    # Pods for testing
  end

  use_native_modules!
end

target 'x-tvOS' do
  # Pods for x-tvOS

  target 'x-tvOSTests' do
    inherit! :search_paths
    # Pods for testing
  end

end


Any solution?

**Comment by tjengland** [comment]
https://github.com/react-native-community/react-native-maps/blob/master/docs/installation.md

which RN version?

also - doesn't look like you have everything here

```
  # react-native-maps dependencies
  pod 'react-native-maps', path: rn_maps_path
  # pod 'react-native-google-maps', path: rn_maps_path  # Uncomment this line if you want to support GoogleMaps on iOS
  # pod 'GoogleMaps'  # Uncomment this line if you want to support GoogleMaps on iOS
  # pod 'Google-Maps-iOS-Utils' # Uncomment this line if you want to support GoogleMaps on iOS
```

**Comment by scotbond** [comment]
Please upgrade to react-native-streetview v0.2.1 which contains proper podspec. that should sort this out. please update so I can close this issue.

**Comment by Alaaeldin-as1405124** [comment]
#Fixed @react-native-streetview v0.2.1 
Thank you very much.
